test/test_reactivities_bosch_hale.py

```diff
@@ -0,0 +1,8 @@
+import fusion_toolbox.reaction_rate.bosch_hale as bh
+import numpy.testing as npt
+
+def test_DTn_reactivity():
```

Comment:
You can use the parametrize decorator we learned about, where `test` would be the temperature in keV and the `expected` would be the reactivity